tests/test_modeling_gpt2.py

```diff
@@ -37,6 +37,7 @@
 class GPT2ModelTest(ModelTesterMixin, unittest.TestCase):
 
     all_model_classes = (GPT2Model, GPT2LMHeadModel, GPT2DoubleHeadsModel) if is_torch_available() else ()
+    all_generative_model_classes = (GPT2LMHeadModel,) if is_torch_available() else ()
```

Comment:
With new testing design these changes would have to be made to a model file to add it to the language generation testing. Is that ok? @LysandreJik 

Could also move some variables to a generic `prepare_language_generation()` fn in `test_modeling_common.py`, such as `num_beams`, `max_length` and `num_return_sequences`, since there are the same everywhere. I feel like the token variables should stay in the respective model files since they differ very strongly and are part of the model config. What do you think? @LysandreJik

Reply:
Yes, I agree with you.